cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
+// ***********************************************
+// This example commands.ts shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
```

Comment:
What makes commands different from being a child vs dual command? I guess it comes back to me not understanding the terminology. You would add a child command and just specifying 'add' again it's considered a dual command?

Reply:
A dual command is used to define a middle process between a parent and child and used to represent a sub-process (based on my understanding). So housed under a parent, but can have children of its own. I think for our use case, we won't really need to worry about those too much.